ods_ci/tests/Tests/400__ods_dashboard/401__ods_dashboard.robot

```diff
@@ -129,17 +130,16 @@ Verify CSS Style Of Getting Started Descriptions
     Capture Page Screenshot    get_started_sidebar.png
     Verify Jupyter Card CSS Style
 
-Verify Documentation Link HTTP Status Code
-    [Documentation]    It verifies the documentation link present in question mark and
+Verify Documentation Links HTTP Status Code
+    [Documentation]    It verifies the documentation links present in question mark and
     ...    also checks the RHODS dcoumentation link present in resource page.
+    ...    ProductBug: RHOAIENG-11451
     [Tags]    Sanity    Tier1
     ...       ODS-327    ODS-492
-    ...       AutomationBugOnODH
-    # TODO: In ODH the expected Docs links are:
-    # https://opendatahub.io/community
-    # https://opendatahub.io/docs
+    ...       ExcludeOnODH
     ${links}=  Get RHODS Documentation Links From Dashboard
-    Documentation Links Should Be Equal To The Expected Ones   actual_links=${links}  expected_links=${DOC_LINKS_EXP}
```

Comment:
Is this still used somewhere else? Do we want to remove this keyword?

Reply:
It is not used, but I wanted to keep this keyword if it will be required to be used later. 
wdyt, should we remove it ?